drivers/lora/Kconfig.sx12xx

```diff
@@ -42,6 +44,12 @@ config LORA_SX126X
 	help
 	  Enable LoRa driver for Semtech SX1261 and SX1262.
 
+config LORA_STM32WL_SUBGHZ_RADIO
```

Comment:
Wouldn't LORA_STM32WL be enough as a symbol name? It isn't really important, of course.

Reply:
Yeah that was what I originally picked as well but it felt wrong to just have the platform name, then I added SUBGHZ as that's how it's referred in the manual, but then I realize that that actually identifies the internal SPI port, and ended up here. I'm fine with anything really... @erwango is there a convention we should follow here?

Reply:
Unfortunately, I don't think we have any convention on this